hawkbit-ui/pom.xml

```diff
@@ -252,6 +252,18 @@
          <artifactId>spring-boot-configuration-processor</artifactId>
          <optional>true</optional>
       </dependency>
+      <dependency>
+         <groupId>com.github.gwtd3</groupId>
+         <artifactId>gwt-d3-api</artifactId>
+         <version>1.2.0</version>
```

Comment:
would be good to move this dependency into the parent-pom to the `<dependencyManagement>`  where all dependencies of hawkBit are declared